assets/js/components/Work/UIWorkCardItem.jsx

```diff
@@ -1,64 +1,78 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { setVisibilityClass, formatDate } from "../../services/helpers";
+import { IIIFProvider, IIIFContext } from "../../components/IIIF/IIIFProvider";
 
-const WorkCardItem = ({ work }) => {
+const WorkCardItem = ({
+  id,
+  representativeImage,
+  title,
+  descriptiveMetadata,
+  workType,
+  visibility,
+  published,
+  accessionNumber,
+  fileSets,
```

Comment:
We don't need all the `fileSets` and their contents, right?  This could get real heavy if there are 1000 or more `fileSets` per Work.  Let's just send in the total number of fileSets.

Reply:
This is a good one, I'll make this change.